sql/core/src/main/scala/org/apache/spark/sql/expressions/UserDefinedFunction.scala

```diff
@@ -38,114 +38,106 @@ import org.apache.spark.sql.types.DataType
  * @since 1.3.0
  */
 @Stable
```

Comment:
yea actually I was wondering about the same thing.

Reply:
I'd go ahead and leave the Since version. The API is essentially unchanged, though there are some marginal breaking compile time changes. But same is true of many things we are changing in 3.0. I've tagged the JIRA with `release-notes` and will add a blurb about the change.

Reply:
It's not a new API anyway, it will be weird to change since to 3.0.

Reply:
Got it. Thank you, @HyukjinKwon , @srowen , @cloud-fan .